search/searcher/search_phrase.go

```diff
@@ -31,7 +32,24 @@ type PhraseSearcher struct {
 	initialized  bool
 }
 
-func NewPhraseSearcher(indexReader index.IndexReader, mustSearcher *ConjunctionSearcher, terms []string) (*PhraseSearcher, error) {
+func NewPhraseSearcher(indexReader index.IndexReader, terms []string, field string, options search.SearcherOptions) (*PhraseSearcher, error) {
+	options.IncludeTermVectors = true
+	termSearchers := make([]search.Searcher, 0)
+	for _, term := range terms {
+		if term != "" {
+			ts, err := NewTermSearcher(indexReader, term, field, 1.0, options)
+			if err != nil {
```

Comment:
ya, we always had that problem, suspect we'll need your deferred closure trampoline megajump trick